### ISSUE TITLE
CSS Shaders - 

### ISSUE THREAD

**Issue by Fyrd** [Fyrd/caniuse#1353]
CSS Shaders - http://www.adobe.com/devnet/html5/articles/css-shaders.html

---------
##### This issue was imported from [Google Moderator](https://www.google.com/moderator/)
Moderator votes: __+4__
Issue added by __L__ on __2011-10-23__
To vote this issue up or down, simply include `+1` or `-1` in your comment.

**Comment by Schweinepriester** [comment]
Probably already covered by <http://caniuse.com/#feat=css-filter-function>?

From <https://developers.google.com/web/updates/2013/03/Introduction-to-Custom-Filters-aka-CSS-Shaders>:

> Custom Filters, or CSS Shaders as they used to be called …

**Comment by yukulele** [comment]
https://developers.google.com/web/updates/2013/03/Introduction-to-Custom-Filters-aka-CSS-Shaders

**Comment by yukulele** [comment]
only supported by chromium based browsers ?

**Comment by strarsis** [comment]
But where can I find out whether I can already use custom CSS shaders (`filter` `url(...)`)?

**Comment by yukulele** [comment]
This feature seems to be abandoned.

From https://drafts.fxtf.org/filter-effects/#changes:

> The following significant changes were made since the [26 November 2013 Working Draft](https://www.w3.org/TR/2013/WD-filter-effects-1-20131126/).
> 
> Removed Custom Filters.
</cite>

Spec:
https://drafts.fxtf.org/custom/

**Comment by SebastianZ** [comment]
Yes, Custom Filters aka CSS Shaders are not a thing anymore, unfortunately. So I suggest to close this issue.

Sebastian
